python/delta/tables.py

```diff
@@ -581,6 +581,21 @@ def upgradeTableProtocol(self, readerVersion: int, writerVersion: int) -> None:
                              type(writerVersion))
         jdt.upgradeTableProtocol(readerVersion, writerVersion)
 
+    def addFeatureSupport(self, feature: str) -> None:
```

Comment:
Could we add `@since(3.3)  # type: ignore[arg-type]` like other Python APIs?

Reply:
Oops thanks for pointing out. Fixed.